src/app/components/lb/lb.component.ts

```diff
@@ -24,22 +26,22 @@ export class LbComponent extends EditionlevelSusceptible {
           return true;
         }
         // Otherwise:
-        // - in diplomatic and interpretative edition, if the text has at least one line, those are show as block items
-        // - in critical edition lines are always shown as inline items
+        // - in diplomatic and interpretative edition, if the text has at least one line,
+        // those are show as block items, unless current text flow is verses
+        // - in critical editionm lines are always shown as inline items, unless current text flow is prose
         switch (this.editionLevel) {
           case 'diplomatic':
           case 'interpretative':
-            return hasLines;
+            return this.textFlow === 'verses' ? false : hasLines;
```

Comment:
this is `this.textFlow === 'verses' || hasLines`

Reply:
Don't think this will work. If textFlow is set as "verses" it doesn't matter if it has lines or not, it should return false. Otherwise, it depends on hasLines